src/common/unmarkedWsNoticeDetails/UnmarkedWsNoticeDetails.tsx

```diff
@@ -0,0 +1,149 @@
+    winterStorageAreaName: string;
+    winterStorageArea: string;
+  };
+  id: string;
+  status: ApplicationStatus;
+  summaryInformation?: SummaryInformation;
+}
+
+const UnmarkedWsNoticeDetails = ({
+  applicant,
+  createdAt,
+  status,
+  boatType,
+  boatRegistrationNumber,
+  boatWidth,
+  boatLength,
+  boatDraught,
+  boatWeight,
+  boatName,
+  boatModel,
+  choice,
+  summaryInformation,
+}: UnmarkedWsNoticeDetailsProps) => {
+  const { t, i18n } = useTranslation();
+
+  return (
+    <Grid colsCount={3}>
+      <div>
+        <Section title={t('unmarkedWsNotices.noticeDetails.notice').toUpperCase()}>
+          <LabelValuePair label={'Alue'} value={choice.winterStorageArea} />
+          <LabelValuePair
+            label={t('applicationList.applicationDetails.receivedDate')}
+            value={formatDate(createdAt, i18n.language, true)}
+          />
+          <LabelValuePair
+            label={t('applicationList.applicationDetails.status')}
+            value={t(APPLICATION_STATUS[status]?.label)}
+          />
+        </Section>
+        {applicant &&
+          ('organization' in applicant ? (
+            <OrganizationCustomerDetails
+              {...applicant}
+              title={t('unmarkedWsNotices.noticeDetails.ownerInformation').toUpperCase()}
+            />
+          ) : (
+            <PrivateCustomerDetails
+              {...applicant}
+              title={t('unmarkedWsNotices.noticeDetails.ownerInformation').toUpperCase()}
+            />
+          ))}
+      </div>
+      <div>
+        <Section title={t('applicationList.applicationDetails.boatInfo')}>
+          <LabelValuePair label={t('applicationList.applicationDetails.boatType')} value={boatType} />
+          <LabelValuePair
+            label={t('applicationList.applicationDetails.registrationNumber')}
+            value={boatRegistrationNumber}
+          />
+        </Section>
+        <Section>
+          <LabelValuePair
+            label={t('applicationList.applicationDetails.boatWidth')}
+            value={formatDimension(boatWidth, i18n.language)}
+          />
+          <LabelValuePair
+            label={t('applicationList.applicationDetails.boatLength')}
+            value={formatDimension(boatLength, i18n.language)}
+          />
+          <LabelValuePair
+            label={t('applicationList.applicationDetails.boatDepth')}
+            value={formatDimension(boatDraught, i18n.language)}
+          />
+          <LabelValuePair
+            label={t('applicationList.applicationDetails.boatWeight')}
+            value={formatWeight(boatWeight, i18n.language)}
+          />
+        </Section>
+        <Section>
+          <LabelValuePair label={t('applicationList.applicationDetails.boatName')} value={boatName} />
+          <LabelValuePair label={t('applicationList.applicationDetails.boatBrand')} value={boatModel} />
+        </Section>
+        {summaryInformation && Object.values(summaryInformation).some((value) => value) && (
```

Comment:
I this the `values` check necessary? can there be such objects with no values at all?

Reply:
This `Object.values(...).some(...)` checks if any of the values in `summaryInformation` is `true`. This is used to hide the empty section if the applicant has not chosen any of the newsletters.

Just a thought, contrary to the schema `summaryInformation` probably exists for all notices.

Reply:
ok, fair enough 👍